README.md

```diff
@@ -1,12 +1,97 @@
-# Git Repository Template
+<p align="center">
+  <a href="https://nimblehq.co/">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="https://assets.nimblehq.co/logo/dark/logo-dark-text-320.png">
+      <img alt="Nimble logo" src="https://assets.nimblehq.co/logo/light/logo-light-text-320.png">
+    </picture>
+  </a>
+</p>
 
-Project repository template to set up all public projects at [Nimble](https://nimblehq.co/)
+<h3 align="center">Mulesoft GitHub Actions</h3>
+<p>A collection of composite GitHub actions designed specifically for Mulesoft integration projects. These actions provide pre-built automation steps and workflows to streamline development, testing, and deployment processes.</p>
```

Comment:
```suggestion
<p>A collection of composite GitHub actions designed specifically for Mulesoft projects. These actions provide pre-built automation steps and workflows to streamline development, testing, and deployment processes.</p>
```

Reply:
Changed in https://github.com/nimblehq/mulesoft-actions/pull/1/commits/18de2c0137ad2dddfcc650d4cc6353c169c92350